staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -50,9 +52,9 @@ var (
 	requestCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "apiserver_request_total",
-			Help: "Counter of apiserver requests broken out for each verb, group, version, resource, scope, component, client, and HTTP response contentType and code.",
+			Help: "Counter of apiserver requests broken out for each verb, dry run value, group, version, resource, scope, component, client, and HTTP response contentType and code.",
 		},
-		[]string{"verb", "group", "version", "resource", "subresource", "scope", "component", "client", "contentType", "code"},
+		[]string{"verb", "dryRun", "group", "version", "resource", "subresource", "scope", "component", "client", "contentType", "code"},
```

Comment:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/instrumentation.md for guidelines, prefer `dry_run`

Reply:
I'll update this, thanks. I only put "dryRun" to conform with the existing "contentType" label

Reply:
I agree that consistency should matter more here.

Reply:
It looks terrible to have both, but you should probably do what sig instrumentation wants; leave a comment, though.